Command/DocumentCommand.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace EMS\CoreBundle\Command;
+
+use Elasticsearch\Client;
+use EMS\CoreBundle\Entity\ContentType;
+use EMS\CoreBundle\Exception\CantBeFinalizedException;
+use EMS\CoreBundle\Exception\NotLockedException;
+use EMS\CoreBundle\Helper\Archive;
+use EMS\CoreBundle\Service\ContentTypeService;
+use EMS\CoreBundle\Service\DataService;
+use EMS\CoreBundle\Service\DocumentService;
+use Monolog\Logger;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+
+class DocumentCommand extends Command
+{
+    const ARGUMENT_CONTENTTYPE = 'contentTypeName';
+    protected static $defaultName = 'ems:make:document';
+
+    /** @var DocumentService */
+    private $documentService;
+    /** @var ContentTypeService */
+    private $contentTypeService;
+    /** @var DataService */
+    private $dataService;
+    /** @var Logger */
+    private $logger;
+    /** @var Client */
+    private $client;
+    /** @var SymfonyStyle */
+    private $io;
+    /** @var ContentType */
+    private $contentType;
+    /** @var string */
+    private $archiveFilename;
+    /** @var bool */
+    private $ready;
+
+    public function __construct(Logger $logger, Client $client, ContentTypeService $contentTypeService, DocumentService $documentService, DataService $dataService)
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->documentService = $documentService;
+        $this->dataService = $dataService;
+        $this->dataService = $dataService;
+        $this->logger = $logger;
+        $this->client = $client;
+        $this->ready = false;
+        parent::__construct();
+    }
+    
+    protected function configure()
+    {
+        $this
+            ->setDescription('Import json files from a zip file as content type\'s documents')
+            ->addArgument(
+                self::ARGUMENT_CONTENTTYPE,
+                InputArgument::REQUIRED,
+                'Content type name to import into'
+            )
+            ->addArgument(
+                'archive',
```

Comment:
use constant for arguments (as we will fetch them using the getArgument) --> sorry for that.

Reply:
No constants needed for the options, as we fetch them as a list!

Reply:
bfe9268fd8233786fbe4b0c110862ca8290d87d2